src/docs/rts-new.md

```diff
@@ -0,0 +1,170 @@
+# Interaction with timestamps
+
+The main thing RTS is concerned with are the durable and stable
+timestamps. The high-level details are covered above, what follows is a more
+detailed view of how we use these timestamps.
+
+We read `txn_global` for pinned and stable timestamps at the start of RTS,
```

Comment:
Is `pinned` discussed above? Because just reading this I expected you to talk about durable and stable timestamps from the previous paragraph. Now durable is not mentioned but pinned is introduced.

I suggest not referring to `txn_global` by structure variable name. It is not mentioned anywhere else in the docs. I suggest rewording to `We read the global pinned and stable timestamps...`

Reply:
Thanks, fixed in 4f053fe.